app/components/agenda/list.hbs

```diff
@@ -18,40 +18,42 @@
       </tr>
     </thead>
 
-    <SortableObjects @tagName={{"tbody"}} @sortEndAction={{this.sort}} @sortableObjectList={{@agendapunten}} enableSort={{true}} useSwap={{true}} inPlace={{true}}>
+    <SortableObjects @tagName={{"tbody"}} @sortEndAction={{@sort}} @sortableObjectList={{@agendapunten}} enableSort={{true}} useSwap={{true}} inPlace={{true}}>
       {{#each @agendapunten as |agendapunt|}}
-        {{#draggable-object tagName="tr" content=agendapunt isSortable=true}}
-          {{!--  <td class="u-align-right">1</td> --}}
-          <td>
-            {{#wu-link-button
-              isTiny=true
-              isLink=true
-              isNarrow=true
-              onClick=null}}
-              <span style="cursor:pointer">
-                <i class="fa fa-bars"></i>
-              </span>
-            {{/wu-link-button}}
-          </td>
-          <td>
-            <p>{{agendapunt.titel}}</p>
-          </td>
-          <td>
-            {{if agendapunt.geplandOpenbaar (t "manageAgendaZittingModal.geplandOpenbaarTrueLabel") (t "manageAgendaZittingModal.geplandOpenbaarFalseLabel")}}
-          </td>
-          <td>
-            {{#wu-button-group}}
-              <WuButton
-                @isSmall={{true}}
-                @isAlt={{true}}
-                @isNarrow={{true}}
-                @label={{"Bewerk"}}
-                @onClick={{fn @edit agendapunt}}>
-                <i class="vi vi-edit"></i> {{t "manageAgendaZittingModal.editItemButton"}}
-              </WuButton>
-            {{/wu-button-group}}
-          </td>
-        {{/draggable-object}}
+        {{#unless agendapunt.isDeleted}}
+          {{#draggable-object tagName="tr" content=agendapunt isSortable=true}}
+            {{!--  <td class="u-align-right">1</td> --}}
+            <td>
+              {{#wu-link-button
+                isTiny=true
+                isLink=true
+                isNarrow=true
+                onClick=null}}
+                <span style="cursor:pointer">
+                  <i class="fa fa-bars"></i>
+                </span>
+              {{/wu-link-button}}
+            </td>
+            <td>
+              <p>{{agendapunt.titel}}</p>
+            </td>
+            <td>
+              {{if agendapunt.geplandOpenbaar (t "manageAgendaZittingModal.geplandOpenbaarTrueLabel") (t "manageAgendaZittingModal.geplandOpenbaarFalseLabel")}}
+            </td>
+            <td>
+              {{#wu-button-group}}
+                <WuButton
```

Comment:
same as above, although this is using octane syntax, somewhat sloppy mixing them ;)